schedule/yast/sle/flows/default_s390x_zfcp.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I don't see the reason to create a additional file for zfcp, this is just a type of disk, we should overwrite on top of the specific arch/virtualization default file for s390x:
schedule/yast/sle/flows/default_s390x_zvm.yaml
Please, revert this in a follow-up PR.

Reply:
> I don't see the reason to create a additional file for zfcp, this is just a type of disk, we should overwrite on top of the specific arch/virtualization default file for s390x: schedule/yast/sle/flows/default_s390x_zvm.yaml Please, revert this in a follow-up PR.

OK.